### PR TITLE
Remove contributions code from frontend

### PR DESCRIPTION
**Pull request by tomrf1**
We have now completed migrating targeting/rendering of contributions messages to the [dotcom-components service](https://github.com/guardian/support-dotcom-components/blob/main/docs/architecture.md).

This PR removes a _lot_ of unused code relating to the epic/banner.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/contributions-liveblog-utilities.js [hunk 1]
```diff
@@ -1,160 +1,29 @@
-import { logView } from 'common/modules/commercial/acquisitions-view-log';
 import $ from 'lib/$';
-import mediator from 'lib/mediator';
-import { elementInView } from 'lib/element-inview';
-import fastdom from 'lib/fastdom-promise';
-import {submitInsertEvent, submitViewEvent} from "common/modules/commercial/acquisitions-ophan";
 import { mountDynamic } from "@guardian/automat-modules";
 
-let isAutoUpdateHandlerBound = false;
-const INSERT_EPIC_AFTER_CLASS = 'js-insert-epic-after';
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/contributions-liveblog-utilities.js:10]: No longer support manually placing an epic in a specific spot, as this isn't being used.
So now we only place a single epic in a randomly chosen spot

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 2]
```diff
@@ -1,266 +1,20 @@
-import { isAbTestTargeted } from 'common/modules/commercial/targeting-tool';
 import {
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js:1]: this file is now just a handful of helper functions

File: static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js [hunk 3]
```diff
@@ -5,21 +5,19 @@ import {
...
     incrementMvtCookie,
 } from 'common/modules/analytics/mvt-cookie';
 import { setGeolocation, getSync as geolocationGetSync } from 'lib/geolocation';
-import config from 'lib/config';
 import { clearParticipations } from 'common/modules/experiments/ab-local-storage';
-import { isBlocked } from 'common/modules/commercial/membership-engagement-banner-block';
 import { pageShouldHideReaderRevenue } from 'common/modules/commercial/contributions-utilities';
+import userPrefs from 'common/modules/user-prefs';
+
+const lastClosedAtKey = 'engagementBannerLastClosedAt';
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js:19]: moved from the now-deleted `membership-engagement-banner.js`

File: static/src/stylesheets/module/reader-revenue/_epic.scss [hunk 4]
```diff
@@ -1,51 +1,5 @@
 @import '../experiments/svg';
```
Comment (tomrf1) [inline static/src/stylesheets/module/reader-revenue/_epic.scss:1]: a tiny amount of this css is still used for a message on the 'Today in Focus' page and the adblock message

**Review by shtukas — APPROVED** [review]

**Pull request merged**